### ISSUE TITLE
vmware-iso on remote esx5 with post-processor fails with invalid disk type

### ISSUE THREAD

**Issue by rvalente** [hashicorp/packer#3632]
- Packer Version: 0.10.1
- Host platform: Ubuntu Linux 14.04.4

The VM build successfully and ships the OVA back to the Jenkins build node. Once there it attempts to upload it to 4 vCenters globally and fails on the first one with the following error.

```
Build 'vmware-iso' errored: 1 error(s) occurred:

* Post-processor failed: Failed: exit status 1
Stdout: Opening OVA source:trusty64-tmp/trusty64-tmp.ova/trusty64-tmp.ova
The manifest validates
Opening VMX target: "vi://USERNAME%40vsphere.local:REDACTED@REDACTED/SITE/host/CLUSTER/Resources/Bronze"
Error: Invalid target disk adapter type: "seSparse"
Deleting directory tree below: "vi:
Completed with errors
```

I get the same error if I do not set a disk_mode, but replace `seSparse` with `thick` or `thin`.

I have also tried this with `format: vmx` which also failed.

If I take the output from the debug log and run ovftool manually it works fine:

```
ovftool --noSSLVerify=true --acceptAllEulas --name="trusty64-vmx" --overwrite \
    --datastore="DS" --diskMode="seSparse" --network="NETWORK" \
    --vmFolder="Templates" trusty64-tmp.ova/trusty64-tmp.ova \
"vi://USERNAME%40vsphere.local:REDACTED@REDACTED/SITE/host/CLUSTER/Resources/Bronze"
```

**Comment by rickard-von-essen** [comment]
There is a unreleased fix on master regarding vSphere #3321. You can try and build from source and see if it solves your issue but it looks like a slightly different issue.

**Comment by SwampDragons** [comment]
Closing this issue since the last comment was 4 years ago and indicates that this was probably fixed on master. If this is still an issue, we can reopen.
